### DISCUSSION TITLE
Can't connect from just tray icon when using flatpak

### DISCUSSION THREAD

**Discussion by MANICX100** [rustdesk/rustdesk#8517] [General]
### Bug Description

If I run
flatpak run com.rustdesk.RustDesk/x86_64/master --tray
I get the tray icon but no connection whereas if I don't use tray it works perfectly likewise tray works when using deb

### How to Reproduce

install the flatpak
run
flatpak run com.rustdesk.RustDesk/x86_64/master --tray
try and connect

### Expected Behavior

it works in the background

### Operating system(s) on local side and remote side

Sparky Linux (Debian 12)

### RustDesk Version(s) on local side and remote side

1.2.6

### Screenshots

![ksnip_20240627-200642](https://github.com/rustdesk/rustdesk/assets/5303241/2b2db6cd-14ab-41b2-aa7e-07a38a408b49)

### Additional Context

Terminal says
flatpak run com.rustdesk.RustDesk/x86_64/master --tray
Gtk-Message: 20:06:07.816: Failed to load module "xapp-gtk3-module"
which: no systemctl in (/app/bin:/usr/bin)

systemctl does exist
whereis systemctl
systemctl: /usr/bin/systemctl /usr/share/man/man1/systemctl.1.gz

**Comment by rustdesk** [comment]
dup: https://github.com/rustdesk/rustdesk/discussions/8098
